test/Aqua.jl

```diff
@@ -4,6 +4,6 @@ using Aqua
   Aqua.test_all(
     Hecke;
     ambiguities=false,      # TODO: fix ambiguities
-    piracies=false          # TODO: fix piracy
+    piracies=true          # TODO: fix piracy
```

Comment:
```suggestion
    piracies=false          # TODO: fix piracy
```
This change shouldn't be in this PR.

Reply:
Argh, indeed, that slipped in while I made another experiment. Thank you